### ISSUE TITLE
Implement getsimplexiter

### ISSUE THREAD

**Issue by odow** [JuliaOpt/Gurobi.jl#190]
As reported by https://github.com/JuliaOpt/JuMP.jl/issues/1821.
http://www.gurobi.com/documentation/8.1/refman/itercount.html#attr:IterCount

@stefanobenati, the steps to fix this are

1. Open the file that contains the MathProgBase (MPB) wrapper:
https://github.com/JuliaOpt/Gurobi.jl/blob/master/src/MPB_wrapper.jl

2. Click the pencil "Edit this file" in the top right corner

3. Add the following line to the bottom of the file
```julia
MPB.getsimplexiter(model::GurobiMathProgModel) = get_dblattr(model, "IterCount")
```

4. Repeat steps 1 and 3 locally to test that it works.

5. Scroll to the bottom of the page and follow the instructions to "Propose file change"

**Comment by stefanobenati** [comment]
Thank you odow for your quick response.

I did the change you propose (in my PC), but now the system is not recognizing the instruction 
"get_intattr", that is implemented in "grb_attrs.jl". The error message is


LoadError: MethodError: no method matching get_intattr(::Gurobi.GurobiMathProgModel, ::String)
Closest candidates are: get_intattr(::Gurobi.Model, ::String)

It seems that a Gurobi.Model is different from ::Gurobi.GurobiMathProgModel.
Converting the latter to the former may help? Or other?

**Comment by stefanobenati** [comment]
It happens the same with instruction "get_dblattr".

**Comment by blegat** [comment]
Try `get_dblattr(model.inner, ...`

**Comment by odow** [comment]
Oops, yes, this is my fault. A good example why tests are needed... Want to make a Pull Request now you have it working?

**Comment by stefanobenati** [comment]
I edited the GitHub file (for the first time in my life,🙂) as you
suggested. Any more that I can do?

Il giorno sab 2 feb 2019, 19:45 Oscar Dowson <notifications@github.com> ha
scritto:

**Comment by blegat** [comment]
@stefanobenati Have you done step 5 of @odow [instructions](https://github.com/JuliaOpt/Gurobi.jl/issues/190#issue-405784165) ?

**Comment by odow** [comment]
> I edited the GitHub file (for the first time in my life

Great! Welcome to JuliaOpt!

It looks like you might have missed clicking a button to finish creating the pull request, but don't worry. You can do do from this link:
https://github.com/JuliaOpt/Gurobi.jl/compare/master...stefanobenati:patch-1?expand=1

**Comment by odow** [comment]
Closed by #192
